src/plugins/Dashboard/ActionBrowseTagline.js

```diff
@@ -27,8 +28,9 @@ class ActionBrowseTagline extends Component {
           tabindex="-1"
           type="file"
           name="files[]"
-          multiple="true"
+          multiple={this.props.maxNumberOfFiles !== 1 || !this.props.maxNumberOfFiles}
```

Comment:
not sure what the '|| !this.props.maxNumberOfFiles' is for, according to the left side of the condition, it can be simplified to `!1` which is always false.

Reply:
Yeah, I was trying to make sure `multiple` is `true` if `this.props.maxNumberOfFiles` is `false`, the default value (now default is `null`). I guess just one condition is enough.